### PR TITLE
[CR] AFS Exosuit Updates III

### LINKED ISSUE #69143: [Aftershock] Exosuit power supplies do not function in PSU pocket
### Describe the bug

Exosuit power supplies are not found when stored in the PSU Compartment within the suit.
When a power supply is held in hand the suit can power up and functions properly.
When the power supply is then put into the pocket the suit shuts off and can't be turned on again.

### Attach save file

[Cheneyville-trimmed.tar.gz](https://github.com/CleverRaven/Cataclysm-DDA/files/13265945/Cheneyville-trimmed.tar.gz)

### Steps to reproduce

Create a new character/world
Spawn an Exosuit Frame and Exosuit Power Supply
Wear the frame and pick up the power supply as a handheld item
Power up the Exosuit (Optional)
Insert the Power Supply into the Exosuit
Loose power if powered up, and be unable to turn on the suit

### Expected behavior

Power supplies stored in the pockets of the exosuit should be used to power the suit

### Screenshots

image

### Versions and configuration

- OS: Windows
    - OS Version: 10.0.22621.2428 (22H2)
- Game Version: 20accf2 [64-bit]
- Graphics Version: Tiles
- Game Language: System language []
- Mods loaded: [
    Dark Days Ahead [dda],
    Disable NPC Needs [no_npc_food],
    Portal Storms Ignore NPCs [personal_portal_storms],
    Slowdown Fungal Growth [no_fungal_growth],
    Blaze Industries [blazeindustries],
    Alternative Map Key [alt_map_key],
    sees-player, run, crouch, bleed icons, retrodays [sees_player_retro],
    Aftershock [aftershock]
]

### Additional context

Auxileary power supplies seem to work ok.
If an auxileary power supply is dropped while active, the exosuit remains powered indefinately without drawing power from the main power supply.

### PR DESCRIPTION
**Pull request by Drew4484**
#### Summary
Content "Collection of fixes and updates to Exosuits"

#### Purpose of change
Climate control removal from exosuits leaves pilots too cold even with underlayer.
Power sources lacked USE_UPS and could not have their own power storage charged from another source.

#### Describe the solution

- [x] Adjust underlayer climate control.
- [X] Add USE_UPS to all battery modules.
- [X] Add light exosuit frame: more compact, lighter, lower power consumption. Lacks modular armor and has reduced number of module slots.
- [x] Add more spawns for frames and equipment.
- [x] Remove Underlayer slot from frame pockets.

#### Testing

- [X] UPS fix
- [x] Light frame functionality.
- [X] Itemgroups for light frames.
- [X] Spawns for light frame.
- [x] Review power consumption (consuming too little ATM)

#### Additional context
This resolves #69143.

### CODE REVIEW

**Comment by Drew4484** [comment]
Will resolve #69143

File: data/mods/Aftershock/items/armor/exosuit/exosuit_frame.json [hunk 1]
```diff
@@ -348,6 +349,7 @@
     "description": "This is a standard exosuit frame.  Designed to be modular, it can accept a variety of power supplies, defensive and offensive systems, and utility tools.  Slots exist for the torso, back, arms, and legs.  It is turned on and continually drawing power.  Use it to turn it off.",
     "flags": [
       "USE_UPS",
+      "IS_UPS",
```
Comment (John-Candlebury) [inline data/mods/Aftershock/items/armor/exosuit/exosuit_frame.json:352]: Probably deserves a comment explaining why this is necessary?

File: data/mods/Aftershock/items/armor/exosuit/exosuit_modules.json [hunk 2]
```diff
@@ -127,6 +139,7 @@
       "RECHARGE",
       "WATERPROOF",
       "IS_UPS",
+      "USE_UPS",
```
Comment (John-Candlebury) [inline data/mods/Aftershock/items/armor/exosuit/exosuit_modules.json:142]: Same here for the explanation, given these are the power sources?

**Comment by akrieger** [comment]
> Additional context: WIP

Is it still WIP?

**Comment by Drew4484** [comment]
> > Additional context: WIP
> 
> Is it still WIP?

WIP no more, good to go now.

**Pull request merged**